!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/907/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1267 of commit 9457e7e26b82020d84e18ec71a8f94f7c36d9f4a automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build907/openy-profile # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6484116331367069623.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6941823580628658751.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from https://github.com/ymcatwincities/openy.git
	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:763)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1012)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1043)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1277)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:610)
	at jenkins.scm.SCMCheckoutStrategy.checkout(SCMCheckoutStrategy.java:86)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:532)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
Caused by: hudson.plugins.git.GitException: Command "git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/*" returned status code 128:
stdout: 
stderr: remote: Counting objects: 74139, done.
remote: Compressing objects:   0% (1/31899)   
remote: Compressing objects:   1% (319/31899)   
remote: Compressing objects:   2% (638/31899)   
remote: Compressing objects:   3% (957/31899)   
remote: Compressing objects:   4% (1276/31899)   
remote: Compressing objects:   5% (1595/31899)   
remote: Compressing objects:   6% (1914/31899)   
remote: Compressing objects:   7% (2233/31899)   
remote: Compressing objects:   8% (2552/31899)   
remote: Compressing objects:   9% (2871/31899)   
remote: Compressing objects:  10% (3190/31899)   
remote: Compressing objects:  11% (3509/31899)   
remote: Compressing objects:  12% (3828/31899)   
remote: Compressing objects:  13% (4147/31899)   
remote: Compressing objects:  14% (4466/31899)   
remote: Compressing objects:  15% (4785/31899)   
remote: Compressing objects:  16% (5104/31899)   
remote: Compressing objects:  17% (5423/31899)   
remote: Compressing objects:  18% (5742/31899)   
remote: Compressing objects:  19% (6061/31899)   
remote: Compressing objects:  20% (6380/31899)   
remote: Compressing objects:  21% (6699/31899)   
remote: Compressing objects:  22% (7018/31899)   
remote: Compressing objects:  23% (7337/31899)   
remote: Compressing objects:  24% (765
```

--- a/openy.installation_types.yml
+++ b/openy.installation_types.yml
@@ -25,7 +25,7 @@ extended:
 
 complete:
   name: Complete/Developer
-  hidden: false
+  hidden: true
   packages:
     - activenet
     - alerts

--- a/openy.packages.yml
+++ b/openy.packages.yml
@@ -50,6 +50,7 @@ editorial:
     - openy_prgf_4c
     - openy_prgf_banner
     - openy_prgf_code_block
+    - openy_prgf_faq
     - openy_prgf_featured_highlights
     - openy_prgf_gallery
     - openy_prgf_grid_content

--- a/openy.profile
+++ b/openy.profile
@@ -221,6 +221,9 @@ function openy_import_content(array &$install_state) {
   $migrate_operations = [];
   $uninstall_operations = [];
   $preset = \Drupal::state()->get('openy_preset') ?: 'complete';
+  if (function_exists('drush_print')) {
+    drush_print(dt('Preset: %preset', ['%preset' => $preset]));
+  }
   $preset_tags = [
     'standard' => 'openy_standard_installation',
     'extended' => 'openy_extended_installation',

--- a/src/Form/ConfigureProfileForm.php
+++ b/src/Form/ConfigureProfileForm.php
@@ -32,7 +32,7 @@ class ConfigureProfileForm extends FormBase {
     $installation_types = Yaml::decode(file_get_contents($path . '/openy.installation_types.yml'));
 
     foreach ($installation_types as $key => $installation_type) {
-      if (!empty($installation_type['hidden'])) {
+      if (!empty($installation_type['hidden']) && !$include_hidden) {
         unset($installation_types[$key]);
       }
     }
@@ -145,7 +145,7 @@ class ConfigureProfileForm extends FormBase {
    *   Presets info.
    */
   private static function getPresetsInfo() {
-    $installation_types = self::getInstallationTypes();
+    $installation_types = self::getInstallationTypes(TRUE);
     $presets_info = [];
     $packages_info = self::getPackages();
 

--- a/src/Form/ConfigureProfileForm.php
+++ b/src/Form/ConfigureProfileForm.php
@@ -13,7 +13,7 @@ use Drupal\Core\Extension\MissingDependencyException;
 class ConfigureProfileForm extends FormBase {
 
   const DEFAULT_PRESET = 'none';
-  const DEFAULT_PRESET_DRUSH = 'standard';
+  const DEFAULT_PRESET_DRUSH = 'complete';
 
   /**
    * {@inheritdoc}
@@ -27,9 +27,15 @@ class ConfigureProfileForm extends FormBase {
    *
    * @return mixed
    */
-  public static function getInstallationTypes() {
+  public static function getInstallationTypes($include_hidden = FALSE) {
     $path = drupal_get_path('profile', 'openy');
     $installation_types = Yaml::decode(file_get_contents($path . '/openy.installation_types.yml'));
+
+    foreach ($installation_types as $key => $installation_type) {
+      if (!empty($installation_type['hidden'])) {
+        unset($installation_types[$key]);
+      }
+    }
 
     return $installation_types;
   }

